index.html

```diff
@@ -1,87 +1,183 @@
+                <div class="myself-3">
+                    <section class="skills-1">
+                        <h4 class="skills-2">LANGUAGE</h4>
+                        <ul class="skills-3">
+                            <li>JavasScript</li>
+                            <li>Ruby</li>
+                            <li>HTML</li>
+                            <li>CSS</li>
+                        </ul>
+                    </section>
+                    <section class="skills-1">
+                        <h5 class="skills-2">FRAMEWORK</h5>
+                        <ul class="skills-3">
+                            <li>Bootstrap</li>
+                            <li>Ruby on Rails</li>
+                            <li>RSPec</li>
+                            <li>CapyBara</li>
+                            <li>Selenium</li>
+                        </ul>
+                    </section>
+                    <section class="skills-1">
+                        <h5 class="skills-2">SKILLS</h5>
+                        <ul class="skills-3">
+                            <li>Codekit</li>
+                            <li>Gitdub</li>
+                            <li>Codepen</li>
+                            <li>Gitlab</li>
+                            <li>Terminal</li>
+                        </ul>
+                    </section>
+                </div>
+            </div>
+        </main>
         <div class="icons-1">
-         <div class="arrow-left"><img src="images/arrow.png" alt="arrow-down"></div>
-            <ul class="icons-2">
-                <li><img src="images/Disabled.png" alt="github-icon"></li>
-                <li><img src="images/Frame.png" alt="LinkedIn-icon"></li>
-                <li><img src="images/angelist.png" alt="angellist-icon"></li>
-                <li><img src="images/Group.png" alt="twitter-icon"></li>
-                <li><img src="images/medium.png" alt="Medium icon"></li>
+            <div class="work-left">
+                <img class="arrow-image left-arrow" src="images/arrow.png" alt="arrow-down">
+            </div>
+            <ul class="links">
+                <li><a href="#"><img src="images/github.png" alt="github-icon"></a></li>
+                <li><a href="#"><img src="images/linkedin.png" alt="linkedIn-icon"></a></li>
+                <li><a href="#"><img src="images/angellist.png" alt="angellist-icon"></a></li>
+                <li><a href="#"><img src="images/twitter.png" alt="twitter-icon"></a></li>
+                <li><a href="#"><img src="images/medium.png" alt="medium-icon"></a> </li>
             </ul>
-            <div><img src="images/arrow.png" alt="arrow-down"></div>
-        </div>
\ No newline at end of file
+            <div class="work-right">
+                <img class="arrow-image right-arrow" src="images/arrow.png" alt="arrow-down">
+            </div>
+        </div>
+        <section class="container-4">
+            <h1 class="title-1">
+                my recent work
+            </h1>
```

Comment:
- [ ] Your "MY RECENT WORK" is not matched with the Figma design. Please work on it. :writing_hand: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![y1](https://user-images.githubusercontent.com/70483835/159107116-836acb80-f8dd-4eee-9d76-bdae9f0be69b.png) | ![f1](https://user-images.githubusercontent.com/70483835/159107164-383d95a2-4efc-41a9-8c3e-d83411fa80f0.png)